services/workspace/src/services/WorkspaceService.ts

```diff
@@ -77,4 +83,28 @@ export class WorkspaceService {
     return this.toPublicQuestionDetailUrl(question);
   }
 
+  public async createQuestion(question: CreateWorkspaceQuestionRequest['question']): Promise<Question> {
+    const createdQuestion = await this.questionRepository.createQuestion({
+      name: question.name,
+      description: question.description,
+      timeLimit: question.timeLimit,
+      memoryLimit: question.memoryLimit,
+      level: question.level,
+      detailPath: '',
+      createdAt: Timestamp.now(),
+      workspace: {
+        connect: {
+          id: question.workspaceId,
+        },
+      },
+    });
+
+    const updatedPathQuestion = await this.questionRepository.updateQuestionById(
+      createdQuestion.id,
+      { detailPath: `workspaces/${question.workspaceId}/questions/${createdQuestion.id}/question.md` },
+    );
+
+    return updatedPathQuestion;
+  }
```

Comment:
Why we don't combine a query into a single one?

Reply:
อยากให้ที่อยู่ของไฟล์มันเกี่ยวโยงกันกับ question id ผมเลยสร้าง question ให้มันพ่น question id ออกมาก่อนอะ